pkg/kubelet/userns/userns_manager.go

```diff
@@ -418,7 +418,7 @@ func (m *UsernsManager) GetOrCreateUserNamespaceMappings(pod *v1.Pod, runtimeHan
 
 	// if the pod requested a user namespace and the runtime doesn't support user namespaces then return an error.
 	if handlerSupportsUserns, err := m.kl.HandlerSupportsUserNamespaces(runtimeHandler); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("runtime does not support user namespaces: %q", err)
```

Comment:
lint complaining this line; you should fix it before retesting.
```suggestion
		return nil, fmt.Errorf("runtime does not support user namespaces: %w", err)
```

Reply:
Fixed, thanks